doc/add-ons.md

```diff
@@ -94,11 +96,11 @@ The easiest way of iterating over an add-on development is by linking the add-on
 
 Now you can build the client and the add-on should be bundled as part of the client code.
 
-There is a typings library available for Typescript development: [Hubs Client TS Types](https://github.com/MozillaReality/hubs-ts-types)
+There is a typings library available for Typescript development: [Hubs Client TS Types](https://github.com/Hubs-Foundation/hubs-ts-types)
 
 ## Currently available add-ons
 
-- [Template add-on](https://github.com/MozillaReality/hubs-template-addon). This add-on serves as a foundation for add-on development.
-- [Duck add-on](https://github.com/MozillaReality/hubs-duck-addon): Replaces the existing `/duck` chat command and refactors it into a Hubs add-on.
-- [Portals add-on](https://github.com/MozillaReality/hubs-portals-addon): Simple portals implementation as an add-on that lets you spawn portals using the `/portal`. It also show how to add key bindings.
-- [Behavior Graphs add-on](https://github.com/MozillaReality/hubs-behavior-graphs-addon/): Initial Behavior Graphs implementation as an add-on
+- [Template add-on](https://github.com/Hubs-Foundation/hubs-template-addon). This add-on serves as a foundation for add-on development.
+- [Duck add-on](https://github.com/Hubs-Foundation/hubs-duck-addon): Replaces the existing `/duck` chat command and refactors it into a Hubs add-on.
+- [Portals add-on](https://github.com/Hubs-Foundation/hubs-portals-addon): Simple portals implementation as an add-on that lets you spawn portals using the `/portal`. It also show how to add key bindings.
+- [Behavior Graphs add-on](https://github.com/Hubs-Foundation/hubs-behavior-graphs-addon/): Initial Behavior Graphs implementation as an add-on
```

Comment:
This section is missing the post-processing add-on.  Could you add it?

Reply:
done